examples/gradle-v2/kotlin-multiplatform-example/build.gradle.kts

```diff
@@ -9,7 +9,12 @@ version = "1.0-SNAPSHOT"
 kotlin {
     jvm()
 
-    js(IR) {
+    js {
+        browser()
+    }
+
+    @OptIn(org.jetbrains.kotlin.gradle.ExperimentalWasmDsl::class)
+    wasmJs {
```

Comment:
Could you please add `wasmWasi` target and declarations in `wasmMain`, so that we can be sure, that we test the original issue from kotlinx-io?
`webMain` and `wasmMain` might behave differently I believe.